styles/main.css

```diff
@@ -0,0 +1,501 @@
+*{
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+html {
+  /* referenced rem sizing here http://snook.ca/archives/html_and_css/font-size-with-rem */
+  font-size: 62.5%;
+  background-image: url("../assets/bg.jpg");
+  background-repeat: no-repeat;
+  background-size: cover;
+
+}
+body {
+  font-family: sans-serif;
+  font-size: 14px; font-size: 1.4rem; /* =14px */
+  background-color: rgba(0, 0, 0, 0.5);
+  padding-bottom: 1rem;
+}
+.wrapper {
+    width: 1024px;
+    /*display: inline-block;*/
+    position: relative;
+    /*overflow: hidden;*/
+    margin: 0 auto;
+    top: 1rem;
+}
+.link__subscribe {
+  text-transform: uppercase;
+  color: rgb(255, 255, 255);
+  text-decoration: none;
+  font-weight: 200;
+  float: right;
+  margin-right: 8rem;
+  margin-top: 2.5rem;
+}
+.link__subscribe:after {
+  /* This is where I pulled special character from: https://brajeshwar.github.io/entities/ */
+  content:"\2022";
+  display: inline-block;
+  font-size: 10px;
+  font-size: 1rem; /* =10px */
+  padding-left: 1rem;
+}
+
+.nav__social {
+  /*border-left: 5px dotted rgb(255, 255, 255);*/
+  position: relative;
+  /*border-radius: 50px;*/
+  float: right;
+}
+.nav__social img{
+  margin-left: 1rem;
+  margin-bottom: 1rem;
+}
+
+.bottom__border{
+  border-bottom: 4px solid rgba(255, 255, 255, 0.5);
+  display: inline-block;
+}
+.bottom__border--horiz{
+  border-bottom: 4px solid rgba(255, 255, 255, 0.5);
+  margin: 0;
+  padding: 0;
+  height: 232px;
+}
+.bottom__border--main{
+  border-bottom: 4px solid rgba(255, 255, 255, 0.5);
+  display: block;
+  box-sizing: content-box;
+}
+.nav__social:before {
+ content:"•••••";
+ font-size: 15px;
+ font-size: 1.5rem;
+ color: rgb(255, 255, 255);
+ transform: rotate(90deg);
+ display: inline-block;
+ letter-spacing: .25rem;
+ vertical-align: top;
+ padding-bottom: 4rem;
+ padding-left: 2rem;
+}
+
+.carousel {
+  position: relative;
+  display: block;
+  width: 100%;
+  height: 360px;
+  background-color: blue;
+  /*justify-content: center;*/
+  overflow: hidden;
+  border: 8px solid rgb(255, 255, 255);
+}
+.carousel ol li img:first-child{
```

Comment:
this selector might not need to be so "qualified" could the same thing work with just `.carousel img:first-child` ?